R/df_to_exb.r

```diff
@@ -0,0 +1,340 @@
+#' Dataframe to .exb
+#'
+#' Convert a dataframe to Exmaralda .exb
+#'
+#' @author Valeria Buntiakova <valleriabun@gmail.com>
+#'
+#' @param df an R dataframe object that contains columns named 'tier', 'tier_name', 'content', 'time_start', 'time_end' and 'id'
+#' @param name transcription name
+#' @param outputPath filepath
+#' @param referenced_file a filepath for .wav
+#' @param ud_meta a vector ('key':'value') of meta information
```

Comment:
Is it possible to avoid putting here this file as a table? May be as a separate arguments to `df_to_exb()`? Empty by default... And is this an obligatory information? Add comment to non obligatory arguments.

Reply:
It's neither obligatory nor it has a concrete set of possible keys I could put as separate arguments. A user can put anything they want in there. My initial idea was to leave it empty and let people fill in meta information in Exmaralda's Partitur Editor, but we decided I should try to add it to the function.